atlasdb-config/src/test/java/com/palantir/atlasdb/factory/TransactionManagersTest.java

```diff
@@ -426,6 +432,48 @@ public void timeLockMigrationReportsReadyIfMigrationNotNeeded() {
         assertTrue(TransactionManagers.timeLockMigrationCompleteIfNeeded(lockAndTimestampServices));
     }
 
+    @Test
+    public void usesTimeLockIfInstallConfigIsEmbeddedButInitialRuntimeConfigContainsTimeLockBlock() {
```

Comment:
`InstallConfigIsUnspecified`

Reply:
Can do - indeed this is the case after this PR